.github/workflows/build-images.yml

```diff
@@ -0,0 +1,48 @@
+
+name: keg-build
+on:
+  push:
+    branches:
+      - ZEN-543-rebuild-workflow
+      - develop
+
+jobs:
+  build-images:
+    name: 'Build Images'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      CLI_BRANCH: ZEN-543-updates-for-workflow
```

Comment:
Seems like this would be better as `${GITHUB_REF##*/}`
* Eventually we want this to run when a PR is submitted, to me it makes sense to make the branch dynamic
  * You probably planned to update it once approved, but it should work even now, so it's one less thing to worry about later

Reply:
Nevermind. This is for the keg-cli, not the keg-hub repo. Disregard this comment.